app/forms/collection_form.rb

```diff
@@ -49,10 +49,18 @@ def persisted?
   attribute :review_enabled, :boolean, default: false
   attribute :email_when_participants_changed, :boolean, default: true
   attribute :email_depositors_status_changed, :boolean, default: true
-end
 
-def duration_must_be_present
-  return if release_duration.present?
+  def duration_must_be_present
+    return if release_duration.present?
+
+    errors.add(:release_duration, 'select a valid duration for release')
+  end
```

Comment:
I was surprised to find this method floating outside the form class, so I shifted it up into the form.